### ISSUE TITLE
Can i make it so the screen turns off after a couple minutes without use, so it dont stay on 24/7?

### ISSUE THREAD

**Issue by SejeAnd** [seho85/klipper-dgus#70]

**Comment by seho85** [comment]
Not easily doable.

See https://github.com/seho85/klipper-dgus/issues/69#issuecomment-1757856067

**Comment by rouvenschandl** [comment]
@SejeAnd Yes, there is a trick, have a look at my fork (as long as my pull-request is still open) there is a customized DWIN_SET file in which the display goes off automatically and comes on again via touch.
